app/experimenter/nimbus-ui/src/lib/config.test.ts

```diff
@@ -0,0 +1,125 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import config, {
```

Comment:
I'd say `config` could be considered a service because it interfaces with browser DOM API to derive app config and needs mocking in tests. But, I wouldn't block over it

Reply:
I can dig it!